lux/utils/utils.py

```diff
@@ -113,6 +113,10 @@ def like_nan(val):
         return math.isnan(val)
 
 
+def like_geo(val):
+    return isinstance(val, str) and val.lower() in {"state", "country"}
```

Comment:
This is a fine implementation for now. Perhaps in the next PR, we can think about making longitude and latitude a geo type also but somehow labelled differently than `state` and `country`. Since that way longitude and latitude would not get lumped with other quantitative types, to be displayed in `Correlation`. 

![image](https://user-images.githubusercontent.com/5554675/109007348-ff236f00-76e6-11eb-89c5-70c995578e9f.png)